src/style.css

```diff
@@ -1 +1,89 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    local(Avenir-Heavy),
+    local(Avenir-Book),
+    url(./fonts/Avenir.ttc) format(ttc);
+}
+
+html {
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar::before {
+  background-image: url(./images/Search.svg);
+  display: block;
+  content: '';
+  position: absolute;
+}
+
+.search-bar-big::before {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar-small::before {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-size: 11px;
+}
+
+.search-bar__input {
+  border: 1px solid #e1e7ed;
+  width: 100%;
+}
+
+.search-bar__input-big {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  line-height: 21.86px;
+  font-size: 16px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  height: 70px;
+  padding-left: 62px;
+  color: #3d4e61;
+}
+
+.search-bar__input-big:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar__input-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 4px 4px 0 #00000040;
+  font-weight: bold;
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input-small {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19.12px;
+  color: #3d4e61;
+
+  text-align: left;
+  height: 42px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  padding-left: 33px;
+}
```

Comment:
```suggestion
.search-bar__input {
  font-family: Avenir, sans-serif;
  background-image: url(./images/Search.svg);
  background-repeat: no-repeat;
  background-position-y: 50%;
  border: 1px solid #e1e7ed;
  width: 100%;
}

.search-bar__input-big {
  background-position-x: 26px;
  background-size: 19px 19px;
  font-weight: 300;
  line-height: 21.86px;
  font-size: 16px;
  box-shadow: 0 1px 8px 0 #3d4e611a;
  height: 70px;
  padding-left: 62px;
  color: #3d4e61;
}

.search-bar__input-big:hover {
  box-shadow: 0 3px 8px 0 #3d4e6133;
}

.search-bar__input-big:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  box-shadow: 0 4px 4px 0 #00000040;
  font-weight: bold;
  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
}

.search-bar__input-small {
  background-position-x: 13px;
  background-size: 11px;
  
  font-size: 14px;
  font-weight: 400;
  line-height: 19.12px;
  color: #3d4e61;

  text-align: left;
  height: 42px;
  box-shadow: 0 1px 8px 0 #3d4e611a;
  padding-left: 33px;
}
```
You don`t need to use these pseudo-elements it is redundant, better to use background-image in search-bar__input.

If you have similar properties like font-family then move it to common class for both inputs.

Sizes of background better to set in background-size

Move repeated colors to variables and reuse them